external-crates/move/move-execution/v0/crates/move-bytecode-verifier/src/code_unit_verifier.rs

```diff
@@ -91,8 +90,8 @@ impl<'a> CodeUnitVerifier<'a> {
             None => return Ok(0),
         };
 
-        // create `FunctionView` and `BinaryIndexedView`
-        let function_view = control_flow::verify_function(
+        // create `FunctionContext` and `BinaryIndexedView`
+        let function = control_flow::verify_function(
```

Comment:
The smallest of nits but the renaming here is inconsistent with the renaming in "latest" for this variable -- in "latest" this variable was renamed to `function_context`. So may want to rename here to keep that consistent.